Bottom: 9769845225f90673c1d4d72869950a5e27cc374d
Top:    cb429b5a0822f00968a07b3469b66c5a4786ded4
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-02-21 16:45:07 +0800

Refresh of virtio-net-optimize-small

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index eccde74..bf95016 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -399,7 +399,7 @@ static struct sk_buff *receive_small(struct net_device *dev,
 	unsigned int header_offset = VIRTNET_RX_PAD + xdp_headroom;
 	unsigned int headroom = vi->hdr_len + header_offset;
 	unsigned int buflen = SKB_DATA_ALIGN(GOOD_PACKET_LEN + headroom) +
-		              SKB_DATA_ALIGN(sizeof(struct skb_shared_info));
+			      SKB_DATA_ALIGN(sizeof(struct skb_shared_info));
 	unsigned int delta = 0;
 	len -= vi->hdr_len;
